README.md

```diff
@@ -178,6 +178,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 **[`^        back to top        ^`](#)**
 
   * [Bookie](https://github.com/bookieio/Bookie) - Python based delicious.com replacement. `AGPLv3` `Python`
+  * [dyu/bookmarks](https://github.com/dyu/bookmarks) - Single-threaded/process bookmark app powered by leveldb and uWebSockets.  Supports importing from delicious or chrome ([demo](https://dyuproject.com/bookmarks/) with [bookmarklet](https://dyuproject.com/bookmarklet/)) `Apache` `JNI`
```

Comment:
* Please just use ([Demo](https://dyuproject.com/bookmarks/))
 * Please remove the double space before _Supports_
 * Please capitalize Delicious and Chrome
 * Please add a final `.` after Chrome
 * Please change JNI to Java